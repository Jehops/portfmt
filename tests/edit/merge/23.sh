${PORTEDIT} merge -e 'IGNORE=foo' 23.in | diff -L 23.expected -L 23.actual -u 23.expected -
