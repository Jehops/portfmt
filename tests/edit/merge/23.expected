IGNORE=		foo

.if ${OPSYS} != FreeBSD
IGNORE=	is only for FreeBSD
.elif ${OSVERSION} < 1200502
IGNORE=	will not build on 12.0 due to old toolchain; 11.x untested
.endif
.if ${FLAVOR} == powerpc64le && ${OSVERSION} < 1300116
IGNORE=	will not build on 12.x due to old system
.endif
.include <bsd.port.mk>
